### DISCUSSION TITLE
Add a link to the Admin URL on the site listing

### DISCUSSION THREAD

**Discussion by GJSchaller** [akeeba/panopticon#532] [Ideas]
It would be nice to have a link to the Admin section of a site on the main listing of sites.  The front UI is there (which is good), but occasionally I need to hop to the back end from my overview to check a setting or make some changes - a link to do this would be wonderful.

Thank you!

**Comment by nikosdion** [comment]
I will have to go with a no. There's only so much of real estate on the interface. Having too many things in this confined space makes the interface overwhelming and daunting. The link you're looking for is just two clicks away anyway.

If you don't agree, you can change that on your own installation. Just do a [view template override](https://github.com/akeeba/panopticon/wiki/Advanced-Customisation-(User-Code)#overriding-the-output) – it works just like with Joomla.

**Reply by GJSchaller** [reply]
If space is the main concern, would displaying it as "Front | Back" work?

The main reason I am asking for this is that I have the secret parameter on all of my admin pages (and I don't display the link on the site itself in any way), so having them gathered in one spot would be immensely useful.

I can look into the override for sure, but thought this might be useful for admins who keep a large number of sites and need to quickly hop to the Admin panel from a main listing.

**Reply by nikosdion** [reply]
The URL of the site in the Sites Overview page is displayed merely for disambiguation. Remember, the Title is something you enter, but you may end up with duplicate, generic titles such as “Dev Site” if you're not absolutely religious about your naming. That's why you see the URL, to figure out which “Dev Site” is that.

As I said, you can simply click on the title of the site and you **do see** a link to the administrator **with** the secret URL parameter at the top right. Click on it. Bam! You're in your site's admin.

Removing the disambiguation, creating a confusion about what "Front | Back" means, and messing up the layout because most languages use far more characters than English to express the same meaning (in Greek it would be “Δημόσιο Τμήμα | Διαχειριστής”, more than twice as long as English) just to save a single click is a bad idea. If in your particular use case it is a good idea, I understand, that's why I made Panopticon infinitely customisable.

**Comment by GJSchaller** [comment]
Ah, OK - I see it now, thank you!  Still learning the new UI, so I missed this before.
